Directory.Packages.props

```diff
@@ -0,0 +1,28 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally Condition="'$(ManagePackageVersionsCentrally)' == ''">true</ManagePackageVersionsCentrally>
+
+    <!-- Indicates that a rebuild is required if this file changes. -->
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
I think this happens [automatically](https://github.com/dotnet/sdk/issues/277#issuecomment-257720464), but I haven't found the exact location to prove it

Reply:
https://www.panopticoncentral.net/2019/07/12/msbuildallprojects-considered-harmful/

Reply:
OK I've removed it.